update-i18n

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
```

Comment:
Should this live in locale/ with the Makefile?

Reply:
this was where @lzap put it in his plugins. I am happy to move it, but would prefer consistency

Reply:
Yeah locale/ perhaps could have been better fit, but I also do have ./release-gem big script in root that does the whole job of releasing the gem, that one fits better in root, maybe ./extra. Dunno. Honestly, if you do not plan to use this script, I suggest to exclude it. I rarely use it myself.

Reply:
@ehelms  want me to nuke it?

Reply:
The script sounds useful, I still think it'd be cleaner in locale/ and updated for that path to not clutter top level workspace.